packages/grafana-prometheus/tsconfig.build.json

```diff
@@ -1,4 +1,4 @@
 {
-  "exclude": ["dist", "node_modules", "test", "**/*.test.ts*"],
+  "exclude": ["dist", "node_modules", "__mocks__", "test", "**/*.test.ts*"],
```

Comment:
This excludes the `__mocks__` folder to end up in the build. @bohandley can we run a dry build and see if that works?